etsin_finder/frontend/locale/finnish.js

```diff
@@ -620,6 +637,93 @@ const finnish = {
         addButton: 'Lisää lisenssi',
       },
     },
+    project: {
+      title: 'Projekti',
+      description: 'Projekti jonka tuotoksena aineisto on luotu',
+      addButton: 'Lisää',
+      editButton: 'Muokkaa',
+      tooltipContent: {
+        title: 'Projekti',
+        paragraph: 'Projekti jonka tuotoksena aineisto on luotu',
+      },
+      project: {
+        title: 'Lisää projekti',
+        addButton: 'Lisää',
+        description: 'Projekti jonka tuotoksena aineisto on luotu',
+      },
+      organization: {
+        title: 'Organisaatio *',
+        description: 'Organisaatio(t), jotka ovat olleet osallisena projektissa'
+      },
+      fundingAgency: {
+        title: 'Rahoittajaorganisaatio'
+      },
+      inputs: {
+        title: {
+          label: 'Lisää projektin otsikko',
+          description: 'Projektin nimi, lisää vähintään yksi kieli.',
+          validation: {
+            required: 'Lisää vähintään yksi kieli',
+            string: 'Nimen täytyy olla merkkijono.',
+          },
+        },
+        titleEn: {
+          placeholder: 'Nimi (Engalnti)',
+        },
+        titleFi: {
+          placeholder: 'Nimi (Suomi)',
+        },
+        identifier: {
+          label: 'Yksilöivä tunniste',
+          description: 'Recommended best practice is to identify the resource by means of a string conforming to a formal identification system. An unambiguous reference to the resource within a given context.',
```

Comment:
"Yksiselitteinen viittaus resurssiin tietyssä kontekstissa. On suositeltavaa käyttää virallisen tunnistamisjärjestelmän mukaisesta tunnistetta."

Näin vapaasti suomennettuna. Placeholderihan pitääkin olla niin kamala että sen haluaa vaihtaa mahdollisimman nopeasti ;)